app/data/index.js

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+import { Sequelize, DataTypes } from 'sequelize'
+import { databaseConfig } from '../config/index.js'
+
+const filename = fileURLToPath(import.meta.url)
+const dirname = path.dirname(filename)
+const modelPath = path.join(dirname, 'models')
+
+const db = {}
+
+const dbConfig = databaseConfig.getProperties()
+
+const sequelize = new Sequelize(dbConfig)
+
+const fileExtensionLength = '.js'.length
+
+const files = fs.readdirSync(modelPath)
+  .filter(file => {
+    return (file.indexOf('.') !== 0) && (file !== 'index.js') && (file.slice(-fileExtensionLength) === '.js')
+  })
+
```

Comment:
Consider removing this loop if you do not have any mapped/planning to map relationship between your tables

Reply:
Yep agreed, won't remove it on this branch as Shaun will remove it on his which is the one we're working on next straight after this is merged. Thank you.